package/com.unity.formats.usd/Tests/USD.NET/OverrideTests.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Jeremy Cowles. All rights reserved.
+// Copyright 2023 Unity Technologies. All rights reserved.
```

Comment:
I think this should be added "in addition" to the original copyright notice rather than instead. See section 4.d of the Apache 2.0 license:

> You may add Your own copyright statement to Your modifications and may provide additional or different license terms and conditions for use, reproduction, or distribution of Your modifications, or for any such Derivative Works as a whole, provided Your use, reproduction, and distribution of the Work otherwise complies with the conditions stated in this License.

Reply:
Fixed!